Gemfile.lock

```diff
@@ -3,9 +3,13 @@ GEM
   specs:
 
 PLATFORMS
+  arm64-darwin-24
```

Comment:
Probably don't need this right? Just an artifact from generating this locally on your Mac

Reply:
Good call. Removed and re-built.